src/esp/bindings/GeoBindings.cpp

```diff
@@ -64,6 +66,112 @@ void initGeoBindings(py::module& m) {
            1.0 is a chordal Catmull-Rom spline)",
       "key_points"_a, "num_interpolations"_a, "alpha"_a = .5f);
 
+  // ==== VoxelWrapper ====
+  py::class_<VoxelWrapper, VoxelWrapper::ptr>(m, "VoxelWrapper")
+      // TODO: Replace with magnum binded Strided Array views (Not sure how to
+      // do this yet..)
```

Comment:
I'll look into this the next days -- *ideally*, when it's fixed, you should be able to directly return the typed views.